examples/extension/README.md

```diff
@@ -0,0 +1,12 @@
+# Extension example
```

Comment:
We might want to change this and the folder name to `extension_bundle` since it doesn't cover the plugin extension type.

Reply:
changed by a55feb8. (Sorry, I can't check plugin type extension in my environment.)